--- a/scripts/gcc-wrapper.py	2025-12-01 01:13:26.879546929 +0000
+++ b/scripts/gcc-wrapper.py	2025-12-01 01:13:43.328507824 +0000
@@ -55,7 +55,10 @@ allowed_warnings = set([
     "sched.h:1211", # include/linux/sched.h:1211:1: warning: type qualifiers ignored on function return type
     "halphyrf_8188e_ce.c:2208", # drivers/net/wireless/rockchip_wlan/rtl8189es/hal/phydm/rtl8188e/halphyrf_8188e_ce.c:2208:1: warning: the frame size of 1056 bytes is larger than 1024 bytes
     "halphyrf_8723b_ce.c:2879", # drivers/net/wireless/rockchip_wlan/rtl8723bu/hal/phydm/rtl8723b/halphyrf_8723b_ce.c:2879:1: warning: the frame size of 1056 bytes is larger than 1024 bytes
- ])
+    "workqueue.c:695",  # kernel/workqueue.c:695:24: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
+    "workqueue.c:723",  # kernel/workqueue.c:723:25: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
+    "workqueue.c:745",  # kernel/workqueue.c:745:25: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
+])
 
 # Capture the name of the object file, can find it.
 ofile = None
