--- a/es-app/src/ApiSystem.cpp	2025-08-28 22:34:53.000000000 +0000
+++ b/es-app/src/ApiSystem.cpp	2025-09-15 03:03:46.080187334 +0000
@@ -51,29 +51,29 @@
 #endif
 
 /*
-#define script_config "batocera-config"; // canupdate, overscan enable, overscan disable, storage 'X', storage current, storage list, forgetBT, getRootPassword, lsoutputs
-#define script_overclock "batocera-overclock"; // list, set X
-#define script_upgrade "batocera-upgrade";
-#define script_sync "batocera-sync"; // sync
-#define script_install "batocera-install"; // listDisks, listArchs, install X Y
-#define script_scraper "batocera-scraper";
-#define script_kodi "batocera-kodi";
-#define script_wifi "batocera-wifi"; // scanlist, list, enable X Y, disable
-#define script_bluetooth "batocera-bluetooth"; // trust, list, remove 
-#define script_resolution "batocera-resolution"; // listModes
-#define script_sync "batocera-sync";   // list
-#define script_info "batocera-info"; // --full
-#define script_systems "batocera-systems"; // --filter
-#define script_suport "batocera-support";
-#define script_gameforce "batocera-gameforce"; // buttonColorLed X, powerLed X
-#define script_audio "batocera-audio"; // list, list-profiles, get-profile, set-profile 'X', get, set 'X'
-#define script_bezelproject "batocera-es-thebezelproject"; // list, install X, remove X
-#define script_format "batocera-format"; // listDisks, listFstypes
-#define script_store "batocera-store"; // list, update, refresh, clean-all, install "X", remove "X"
-#define script_preupdategamelists "batocera-preupdate-gamelists-hook";
-#define script_timezones "batocera-timezone"; // get, detect, set "X"
-#define script_padsinfos "batocera-padsinfo";
-#define script_swissknife "batocera-es-swissknife"; // --emukill"
+#define script_config "knulli-config"; // canupdate, overscan enable, overscan disable, storage 'X', storage current, storage list, forgetBT, getRootPassword, lsoutputs
+#define script_overclock "knulli-overclock"; // list, set X
+#define script_upgrade "knulli-upgrade";
+#define script_sync "knulli-sync"; // sync
+#define script_install "knulli-install"; // listDisks, listArchs, install X Y
+#define script_scraper "knulli-scraper";
+#define script_kodi "knulli-kodi";
+#define script_wifi "knulli-wifi"; // scanlist, list, enable X Y, disable
+#define script_bluetooth "knulli-bluetooth"; // trust, list, remove 
+#define script_resolution "knulli-resolution"; // listModes
+#define script_sync "knulli-sync";   // list
+#define script_info "knulli-info"; // --full
+#define script_systems "knulli-systems"; // --filter
+#define script_suport "knulli-support";
+#define script_gameforce "knulli-gameforce"; // buttonColorLed X, powerLed X
+#define script_audio "knulli-audio"; // list, list-profiles, get-profile, set-profile 'X', get, set 'X'
+#define script_bezelproject "knulli-es-thebezelproject"; // list, install X, remove X
+#define script_format "knulli-format"; // listDisks, listFstypes
+#define script_store "knulli-store"; // list, update, refresh, clean-all, install "X", remove "X"
+#define script_preupdategamelists "knulli-preupdate-gamelists-hook";
+#define script_timezones "knulli-timezone"; // get, detect, set "X"
+#define script_padsinfos "knulli-padsinfo";
+#define script_swissknife "knulli-es-swissknife"; // --emukill"
 */
 
 ApiSystem::ApiSystem() { }
@@ -160,7 +160,7 @@
 
 	if (isScriptingSupported(VERSIONINFO)) 
 	{
-		std::string command = "batocera-version";
+		std::string command = "knulli-version";
 		if (extra) 
 			command += " --extra";
 
@@ -218,7 +218,7 @@
 
 bool ApiSystem::setOverscan(bool enable) 
 {
-	return executeScript("batocera-config overscan " + std::string(enable ? "enable" : "disable"));
+	return executeScript("knulli-config overscan " + std::string(enable ? "enable" : "disable"));
 }
 
 bool ApiSystem::setOverclock(std::string mode) 
@@ -226,7 +226,7 @@
 	if (mode.empty())
 		return false;
 
-	return executeScript("batocera-overclock set " + mode);
+	return executeScript("knulli-overclock set " + mode);
 }
 
 // BusyComponent* ui
@@ -241,7 +241,7 @@
 		return std::pair<std::string, int>(std::string("Cannot call update command"), -1);
 	
 	char line[1024] = "";
-	FILE *flog = fopen(Utils::FileSystem::combine(Paths::getLogPath(), "batocera-upgrade.log").c_str(), "w");
+	FILE *flog = fopen(Utils::FileSystem::combine(Paths::getLogPath(), "knulli-upgrade.log").c_str(), "w");
 	while (fgets(line, 1024, pipe)) 
 	{
 		strtok(line, "\n");
@@ -267,14 +267,14 @@
 {
 	LOG(LogDebug) << "ApiSystem::backupSystem";
 
-	std::string updatecommand = "batocera-sync sync " + device;
+	std::string updatecommand = "knulli-sync sync " + device;
 	FILE* pipe = popen(updatecommand.c_str(), "r");
 	if (pipe == NULL)
 		return std::pair<std::string, int>(std::string("Cannot call sync command"), -1);
 
 	char line[1024] = "";
 
-	FILE* flog = fopen(Utils::FileSystem::combine(Paths::getLogPath(), "batocera-sync.log").c_str(), "w");
+	FILE* flog = fopen(Utils::FileSystem::combine(Paths::getLogPath(), "knulli-sync.log").c_str(), "w");
 	while (fgets(line, 1024, pipe)) 
 	{
 		strtok(line, "\n");
@@ -296,14 +296,14 @@
 {
 	LOG(LogDebug) << "ApiSystem::installSystem";
 
-	std::string updatecommand = "batocera-install install " + device + " " + architecture;
+	std::string updatecommand = "knulli-install install " + device + " " + architecture;
 	FILE *pipe = popen(updatecommand.c_str(), "r");
 	if (pipe == NULL)
 		return std::pair<std::string, int>(std::string("Cannot call install command"), -1);
 
 	char line[1024] = "";
 
-	FILE *flog = fopen(Utils::FileSystem::combine(Paths::getLogPath(), "batocera-install.log").c_str(), "w");
+	FILE *flog = fopen(Utils::FileSystem::combine(Paths::getLogPath(), "knulli-install.log").c_str(), "w");
 	while (fgets(line, 1024, pipe)) 
 	{
 		strtok(line, "\n");
@@ -326,13 +326,13 @@
 {
 	LOG(LogDebug) << "ApiSystem::scrape";
 
-	FILE* pipe = popen("batocera-scraper", "r");
+	FILE* pipe = popen("knulli-scraper", "r");
 	if (pipe == nullptr)
 		return std::pair<std::string, int>(std::string("Cannot call scrape command"), -1);
 
 	char line[1024] = "";
 
-	FILE* flog = fopen(Utils::FileSystem::combine(Paths::getLogPath(), "batocera-scraper.log").c_str(), "w");
+	FILE* flog = fopen(Utils::FileSystem::combine(Paths::getLogPath(), "knulli-scraper.log").c_str(), "w");
 	while (fgets(line, 1024, pipe)) 
 	{
 		strtok(line, "\n");
@@ -424,7 +424,7 @@
 	LOG(LogDebug) << "ApiSystem::launchKodi";
 
 	std::string commandline = InputManager::getInstance()->configureEmulators();
-	std::string command = "batocera-kodi " + commandline;
+	std::string command = "knulli-kodi " + commandline;
 
 	ApiSystem::launchExternalWindow_before(window);
 
@@ -471,12 +471,12 @@
 
 bool ApiSystem::enableWifi(std::string ssid, std::string key) 
 {
-	return executeScript("batocera-wifi enable \"" + ssid + "\" \"" + key + "\"");
+	return executeScript("knulli-wifi enable \"" + ssid + "\" \"" + key + "\"");
 }
 
 bool ApiSystem::disableWifi() 
 {
-	return executeScript("batocera-wifi disable");
+	return executeScript("knulli-wifi disable");
 }
 
 std::string ApiSystem::getIpAdress() 
@@ -494,7 +494,7 @@
 {
 	LOG(LogDebug) << "ApiSystem::isWifiAPModeSupported";
 
-	return executeScript("batocera-wifi has_ap_mode");
+	return executeScript("knulli-wifi has_ap_mode");
 }
 
 #ifdef KNULLI
@@ -506,96 +506,96 @@
 
 bool ApiSystem::enableBluetooth()
 {
-	return executeScript("batocera-bluetooth enable 2>&1 >/dev/null");
+	return executeScript("knulli-bluetooth enable 2>&1 >/dev/null");
 }
 
 bool ApiSystem::disableBluetooth()
 {
-	return executeScript("batocera-bluetooth disable");
+	return executeScript("knulli-bluetooth disable");
 }
 
 void ApiSystem::startBluetoothLiveDevices(const std::function<void(const std::string)>& func)
 {
-	executeScript("batocera-bluetooth live_devices", func);
+	executeScript("knulli-bluetooth live_devices", func);
 }
 
 void ApiSystem::stopBluetoothLiveDevices()
 {
-	executeScript("batocera-bluetooth stop_live_devices");
+	executeScript("knulli-bluetooth stop_live_devices");
 }
 
 bool ApiSystem::pairBluetoothDevice(const std::string& deviceName)
 {
-	return executeScript("batocera-bluetooth trust " + deviceName);
+	return executeScript("knulli-bluetooth trust " + deviceName);
 }
 
 bool ApiSystem::connectBluetoothDevice(const std::string& deviceName)
 {
-	return executeScript("batocera-bluetooth connect " + deviceName);
+	return executeScript("knulli-bluetooth connect " + deviceName);
 }
 
 bool ApiSystem::disconnectBluetoothDevice(const std::string& deviceName)
 {
-	return executeScript("batocera-bluetooth disconnect " + deviceName);
+	return executeScript("knulli-bluetooth disconnect " + deviceName);
 }
 
 bool ApiSystem::removeBluetoothDevice(const std::string& deviceName)
 {
-	return executeScript("batocera-bluetooth remove " + deviceName);
+	return executeScript("knulli-bluetooth remove " + deviceName);
 }
 
 bool ApiSystem::scanNewBluetooth(const std::function<void(const std::string)>& func)
 {
-	return executeScript("batocera-bluetooth trust input", func).second == 0;
+	return executeScript("knulli-bluetooth trust input", func).second == 0;
 }
 
 std::vector<std::string> ApiSystem::getPairedBluetoothDeviceList()
 {
-	return executeEnumerationScript("batocera-bluetooth list");
+	return executeEnumerationScript("knulli-bluetooth list");
 }
 
 std::vector<std::string> ApiSystem::getAvailableStorageDevices() 
 {
-	return executeEnumerationScript("batocera-config storage list");
+	return executeEnumerationScript("knulli-config storage list");
 }
 
 std::vector<std::string> ApiSystem::getVideoModes(const std::string output)
 {
   if(output == "") {
-    return executeEnumerationScript("batocera-resolution listModes");
+    return executeEnumerationScript("knulli-resolution listModes");
   } else {
-    return executeEnumerationScript("batocera-resolution --screen \"" + output + "\" listModes");
+    return executeEnumerationScript("knulli-resolution --screen \"" + output + "\" listModes");
   }
 }
 
 std::vector<std::string> ApiSystem::getCustomRunners() 
 {
-	return executeEnumerationScript("batocera-wine-runners");
+	return executeEnumerationScript("knulli-wine-runners");
 }
 
 std::vector<std::string> ApiSystem::getAvailableBackupDevices() 
 {
-	return executeEnumerationScript("batocera-sync list");
+	return executeEnumerationScript("knulli-sync list");
 }
 
 std::vector<std::string> ApiSystem::getAvailableInstallDevices() 
 {
-	return executeEnumerationScript("batocera-install listDisks");
+	return executeEnumerationScript("knulli-install listDisks");
 }
 
 std::vector<std::string> ApiSystem::getAvailableInstallArchitectures() 
 {
-	return executeEnumerationScript("batocera-install listArchs");
+	return executeEnumerationScript("knulli-install listArchs");
 }
 
 std::vector<std::string> ApiSystem::getAvailableOverclocking() 
 {
-	return executeEnumerationScript("batocera-overclock list");
+	return executeEnumerationScript("knulli-overclock list");
 }
 
 std::vector<std::string> ApiSystem::getSystemInformations() 
 {
-	return executeEnumerationScript("batocera-info --full");
+	return executeEnumerationScript("knulli-info --full");
 }
 
 std::vector<BiosSystem> ApiSystem::getBiosInformations(const std::string system) 
@@ -604,7 +604,7 @@
 	BiosSystem current;
 	bool isCurrent = false;
 
-	std::string cmd = "batocera-systems";
+	std::string cmd = "knulli-systems";
 	if (!system.empty())
 		cmd += " --filter " + system;
 
@@ -651,7 +651,7 @@
 
 bool ApiSystem::generateSupportFile() 
 {
-	return executeScript("batocera-support");
+	return executeScript("knulli-support");
 }
 
 std::string ApiSystem::getCurrentStorage() 
@@ -663,7 +663,7 @@
 #endif
 
 	std::ostringstream oss;
-	oss << "batocera-config storage current";
+	oss << "knulli-config storage current";
 	FILE *pipe = popen(oss.str().c_str(), "r");
 	char line[1024];
 
@@ -680,22 +680,22 @@
 
 bool ApiSystem::setStorage(std::string selected) 
 {
-	return executeScript("batocera-config storage " + selected);
+	return executeScript("knulli-config storage " + selected);
 }
 
 bool ApiSystem::setButtonColorGameForce(std::string selected)
 {
-	return executeScript("batocera-gameforce buttonColorLed " + selected);
+	return executeScript("knulli-gameforce buttonColorLed " + selected);
 }
 
 bool ApiSystem::setPowerLedGameForce(std::string selected)
 {
-	return executeScript("batocera-gameforce powerLed " + selected);
+	return executeScript("knulli-gameforce powerLed " + selected);
 }
 
 bool ApiSystem::forgetBluetoothControllers() 
 {
-	return executeScript("batocera-config forgetBT");
+	return executeScript("knulli-config forgetBT");
 }
 
 std::string ApiSystem::getRootPassword() 
@@ -703,7 +703,7 @@
 	LOG(LogDebug) << "ApiSystem::getRootPassword";
 
 	std::ostringstream oss;
-	oss << "batocera-config getRootPassword";
+	oss << "knulli-config getRootPassword";
 	FILE *pipe = popen(oss.str().c_str(), "r");
 	char line[1024];
 
@@ -721,7 +721,7 @@
 
 std::vector<std::string> ApiSystem::getAvailableVideoOutputDevices() 
 {
-	return executeEnumerationScript("batocera-config lsoutputs");
+	return executeEnumerationScript("knulli-config lsoutputs");
 }
 
 std::vector<std::string> ApiSystem::getAvailableAudioOutputDevices() 
@@ -732,7 +732,7 @@
 	return res;
 #endif
 
-	return executeEnumerationScript("batocera-audio list");
+	return executeEnumerationScript("knulli-audio list");
 }
 
 std::string ApiSystem::getCurrentAudioOutputDevice() 
@@ -744,7 +744,7 @@
 	LOG(LogDebug) << "ApiSystem::getCurrentAudioOutputDevice";
 
 	std::ostringstream oss;
-	oss << "batocera-audio get";
+	oss << "knulli-audio get";
 	FILE *pipe = popen(oss.str().c_str(), "r");
 	char line[1024];
 
@@ -767,7 +767,7 @@
 
 	std::ostringstream oss;
 
-	oss << "batocera-audio set" << " '" << selected << "'";
+	oss << "knulli-audio set" << " '" << selected << "'";
 	int exitcode = system(oss.str().c_str());
 
 	Sound::get(":/checksound.ogg")->play();
@@ -783,7 +783,7 @@
 	return res;
 #endif
 
-	return executeEnumerationScript("batocera-audio list-profiles");
+	return executeEnumerationScript("knulli-audio list-profiles");
 }
 
 std::string ApiSystem::getCurrentAudioOutputProfile() 
@@ -795,7 +795,7 @@
 	LOG(LogDebug) << "ApiSystem::getCurrentAudioOutputProfile";
 
 	std::ostringstream oss;
-	oss << "batocera-audio get-profile";
+	oss << "knulli-audio get-profile";
 	FILE *pipe = popen(oss.str().c_str(), "r");
 	char line[1024];
 
@@ -818,7 +818,7 @@
 
 	std::ostringstream oss;
 
-	oss << "batocera-audio set-profile" << " '" << selected << "'";
+	oss << "knulli-audio set-profile" << " '" << selected << "'";
 	int exitcode = system(oss.str().c_str());
 	
 	Sound::get(":/checksound.ogg")->play();
@@ -1095,7 +1095,7 @@
 
 	std::vector<BatoceraBezel> res;
 
-	auto lines = executeEnumerationScript("batocera-es-thebezelproject list");
+	auto lines = executeEnumerationScript("knulli-es-thebezelproject list");
 	for (auto line : lines)
 	{
 		auto parts = Utils::String::splitAny(line, " \t");
@@ -1120,12 +1120,12 @@
 
 std::pair<std::string, int> ApiSystem::installBatoceraBezel(std::string bezelsystem, const std::function<void(const std::string)>& func)
 {
-	return executeScript("batocera-es-thebezelproject install " + bezelsystem, func);
+	return executeScript("knulli-es-thebezelproject install " + bezelsystem, func);
 }
 
 std::pair<std::string, int> ApiSystem::uninstallBatoceraBezel(std::string bezelsystem, const std::function<void(const std::string)>& func)
 {
-	return executeScript("batocera-es-thebezelproject remove " + bezelsystem, func);
+	return executeScript("knulli-es-thebezelproject remove " + bezelsystem, func);
 }
 
 std::string ApiSystem::getMD5(const std::string fileName, bool fromZipContents)
@@ -1566,17 +1566,17 @@
 
 std::vector<std::string> ApiSystem::getWifiNetworks(bool scan)
 {
-	return executeEnumerationScript(scan ? "batocera-wifi scanlist" : "batocera-wifi list");
+	return executeEnumerationScript(scan ? "knulli-wifi scanlist" : "knulli-wifi list");
 }
 
 void ApiSystem::scanWifiNetworks()
 {
-	executeScript("batocera-wifi scanlist &");
+	executeScript("knulli-wifi scanlist &");
 }
 
 std::string ApiSystem::getWifiRoute()
 {
-	std::vector<std::string> result = executeEnumerationScript("batocera-wifi get_route");
+	std::vector<std::string> result = executeEnumerationScript("knulli-wifi get_route");
 
 	if (result.empty() || result[0].empty())
 		return "NOT CONNECTED";
@@ -1658,22 +1658,22 @@
 		executables.push_back("kodi");
 		break;
 	case ApiSystem::WIFI:
-		executables.push_back("batocera-wifi");
+		executables.push_back("knulli-wifi");
 		break;
 	case ApiSystem::BLUETOOTH:
-		executables.push_back("batocera-bluetooth");
+		executables.push_back("knulli-bluetooth");
 		break;
 	case ApiSystem::RESOLUTION:
-		executables.push_back("batocera-resolution");
+		executables.push_back("knulli-resolution");
 		break;
 	case ApiSystem::BIOSINFORMATION:
-		executables.push_back("batocera-systems");
+		executables.push_back("knulli-systems");
 		break;
 	case ApiSystem::DISKFORMAT:
-		executables.push_back("batocera-format");
+		executables.push_back("knulli-format");
 		break;
 	case ApiSystem::OVERCLOCK:
-		executables.push_back("batocera-overclock");
+		executables.push_back("knulli-overclock");
 		break;
 	case ApiSystem::NETPLAY:
 		executables.push_back("7zr");
@@ -1683,52 +1683,52 @@
 		executables.push_back("pdfinfo");
 		break;
 	case ApiSystem::BATOCERASTORE:
-		executables.push_back("batocera-store");
+		executables.push_back("knulli-store");
 		break;
 	case ApiSystem::THEBEZELPROJECT:
-		executables.push_back("batocera-es-thebezelproject");
+		executables.push_back("knulli-es-thebezelproject");
 		break;		
 	case ApiSystem::PADSINFO:
-		executables.push_back("batocera-padsinfo");
+		executables.push_back("knulli-padsinfo");
 		break;
 	case ApiSystem::EVMAPY:
 		executables.push_back("evmapy");
 		break;
 	case ApiSystem::BATOCERAPREGAMELISTSHOOK:
-		executables.push_back("batocera-preupdate-gamelists-hook");
+		executables.push_back("knulli-preupdate-gamelists-hook");
 		break;
 	case ApiSystem::TIMEZONES:
-		executables.push_back("batocera-timezone");
+		executables.push_back("knulli-timezone");
 		break;
 	case ApiSystem::AUDIODEVICE:
-		executables.push_back("batocera-audio");
+		executables.push_back("knulli-audio");
 		break;		
 	case ApiSystem::BACKUP:
-		executables.push_back("batocera-sync");
+		executables.push_back("knulli-sync");
 		break;
 	case ApiSystem::INSTALL:
-		executables.push_back("batocera-install");
+		executables.push_back("knulli-install");
 		break;	
 	case ApiSystem::SUPPORTFILE:
-		executables.push_back("batocera-support");
+		executables.push_back("knulli-support");
 		break;
 	case ApiSystem::UPGRADE:
-		executables.push_back("batocera-upgrade");
+		executables.push_back("knulli-upgrade");
 		break;
 	case ApiSystem::SUSPEND:
 		return (Utils::FileSystem::exists("/usr/sbin/pm-suspend") && Utils::FileSystem::exists("/usr/bin/pm-is-supported") && executeScript("/usr/bin/pm-is-supported --suspend"));
 	case ApiSystem::VERSIONINFO:
-		executables.push_back("batocera-version");
+		executables.push_back("knulli-version");
 		break;
 	case ApiSystem::READPLANEMODE:
 	case ApiSystem::WRITEPLANEMODE:
-		executables.push_back("batocera-planemode");
+		executables.push_back("knulli-planemode");
 		break;
 	case ApiSystem::SERVICES:
-		executables.push_back("batocera-services");
+		executables.push_back("knulli-services");
 		break;
 	case ApiSystem::BACKGLASS:
-		executables.push_back("batocera-backglass");
+		executables.push_back("knulli-backglass");
 		break;
 	}
 
@@ -1788,7 +1788,7 @@
 	ret.push_back("e:\\ DRIVE Z:");
 	return ret;
 #endif
-	return executeEnumerationScript("batocera-format listDisks");
+	return executeEnumerationScript("knulli-format listDisks");
 }
 
 std::vector<std::string> ApiSystem::getFormatFileSystems()
@@ -1799,12 +1799,12 @@
 	ret.push_back("brfs");
 	return ret;
 #endif
-	return executeEnumerationScript("batocera-format listFstypes");
+	return executeEnumerationScript("knulli-format listFstypes");
 }
 
 int ApiSystem::formatDisk(const std::string disk, const std::string format, const std::function<void(const std::string)>& func)
 {
-	return executeScript("batocera-format format " + disk + " " + format, func).second;
+	return executeScript("knulli-format format " + disk + " " + format, func).second;
 }
 
 int ApiSystem::getPdfPageCount(const std::string& fileName)
@@ -1922,7 +1922,7 @@
 
 	LOG(LogDebug) << "ApiSystem::getBatoceraStorePackages";
 
-	auto res = executeEnumerationScript("batocera-store list");
+	auto res = executeEnumerationScript("knulli-store list");
 	std::string data = Utils::String::join(res, "\n");
 	if (data.empty())
 	{
@@ -1991,28 +1991,28 @@
 
 std::pair<std::string, int> ApiSystem::installBatoceraStorePackage(std::string name, const std::function<void(const std::string)>& func)
 {
-	return executeScript("batocera-store install \"" + name + "\"", func);
+	return executeScript("knulli-store install \"" + name + "\"", func);
 }
 
 std::pair<std::string, int> ApiSystem::uninstallBatoceraStorePackage(std::string name, const std::function<void(const std::string)>& func)
 {
-	return executeScript("batocera-store remove \"" + name + "\"", func);
+	return executeScript("knulli-store remove \"" + name + "\"", func);
 }
 
 void ApiSystem::refreshBatoceraStorePackageList()
 {
-	executeScript("batocera-store refresh");
-	executeScript("batocera-store clean-all");
+	executeScript("knulli-store refresh");
+	executeScript("knulli-store clean-all");
 }
 
 void ApiSystem::callBatoceraPreGameListsHook()
 {
-	executeScript("batocera-preupdate-gamelists-hook");
+	executeScript("knulli-preupdate-gamelists-hook");
 }
 
 void ApiSystem::updateBatoceraStorePackageList()
 {
-	executeScript("batocera-store update");
+	executeScript("knulli-store update");
 }
 
 std::vector<std::string> ApiSystem::getShaderList(const std::string& systemName, const std::string& emulator, const std::string& core)
@@ -2129,11 +2129,11 @@
 std::string ApiSystem::getCurrentTimezone()
 {
 	LOG(LogInfo) << "ApiSystem::getCurrentTimezone";
-	auto cmd = executeEnumerationScript("batocera-timezone get");
+	auto cmd = executeEnumerationScript("knulli-timezone get");
 	std::string tz = Utils::String::join(cmd, "");
 	remove_if(tz.begin(), tz.end(), isspace);
 	if (tz.empty()) {
-		cmd = executeEnumerationScript("batocera-timezone detect");
+		cmd = executeEnumerationScript("knulli-timezone detect");
 		tz = Utils::String::join(cmd, "");
 	}
 	return tz;
@@ -2143,7 +2143,7 @@
 {
 	if (tz.empty())
 		return false;
-	return executeScript("batocera-timezone set \"" + tz + "\"");
+	return executeScript("knulli-timezone set \"" + tz + "\"");
 }
 
 std::vector<PadInfo> ApiSystem::getPadsInfo()
@@ -2152,7 +2152,7 @@
 
 	std::vector<PadInfo> ret;
 
-	auto res = executeEnumerationScript("batocera-padsinfo");
+	auto res = executeEnumerationScript("knulli-padsinfo");
 	std::string data = Utils::String::join(res, "\n");
 	if (data.empty())
 	{
@@ -2233,7 +2233,7 @@
 bool ApiSystem::emuKill()
 {
 	LOG(LogDebug) << "ApiSystem::emuKill";
-	return executeScript("batocera-es-swissknife --emukill");
+	return executeScript("knulli-es-swissknife --emukill");
 }
 
 void ApiSystem::suspend()
@@ -2262,7 +2262,7 @@
 
 bool ApiSystem::isPlaneMode()
 {
-	auto res = executeEnumerationScript("batocera-planemode status");
+	auto res = executeEnumerationScript("knulli-planemode status");
 	if (res.size() > 0)
 		return res[0] == "on";
 
@@ -2277,7 +2277,7 @@
 bool ApiSystem::setPlaneMode(bool enable)
 {
 	LOG(LogDebug) << "ApiSystem::setPlaneMode";
-	return executeScript("batocera-planemode " + std::string(enable ? "enable" : "disable"));
+	return executeScript("knulli-planemode " + std::string(enable ? "enable" : "disable"));
 }
 
 std::vector<Service> ApiSystem::getServices()
@@ -2286,7 +2286,7 @@
 
 	LOG(LogDebug) << "ApiSystem::getServices";
 
-	auto slines = executeEnumerationScript("batocera-services list");
+	auto slines = executeEnumerationScript("knulli-services list");
 
 	for (auto sline : slines) 
 	{
@@ -2307,7 +2307,7 @@
 
   LOG(LogDebug) << "ApiSystem::backglassThemes";
 
-  auto slines = executeEnumerationScript("batocera-backglass list-themes");
+  auto slines = executeEnumerationScript("knulli-backglass list-themes");
 
   for (auto sline : slines) 
     {
@@ -2318,7 +2318,7 @@
 
 void ApiSystem::restartBackglass() {
   LOG(LogDebug) << "ApiSystem::restartBackglass";
-  executeScript("/usr/bin/batocera-backglass restart");
+  executeScript("/usr/bin/knulli-backglass restart");
 }
 
 bool ApiSystem::enableService(std::string name, bool enable) 
@@ -2329,9 +2329,9 @@
 
 	LOG(LogDebug) << "ApiSystem::enableService " << serviceName;
 
-	bool res = executeScript("batocera-services " + std::string(enable ? "enable" : "disable") + " " + serviceName);
+	bool res = executeScript("knulli-services " + std::string(enable ? "enable" : "disable") + " " + serviceName);
 	if (res)
-		res = executeScript("batocera-services " + std::string(enable ? "start" : "stop") + " " + serviceName);
+		res = executeScript("knulli-services " + std::string(enable ? "start" : "stop") + " " + serviceName);
 	
 	return res;
 }
