--- a/arch/arm64/boot/dts/rockchip/rk3566-rgb30-linux.dts	2025-10-25 18:51:08.761866198 +0000
+++ b/arch/arm64/boot/dts/rockchip/rk3566-rgb30-linux.dts	2025-10-25 18:48:19.878050361 +0000
@@ -126,7 +126,7 @@
 		compatible = "singleadc-joypad";
 		hw_info = <0x01>;
 
-		joypad-name = "retrogame_joypad";
+		joypad-name = "Powkiddy_RGB30_Controller";
 		joypad-product = <0x1101>;
 		joypad-revision = <0x0100>;
 
--- a/arch/arm64/boot/dts/rockchip/rk3566-rgb30-linux.dts	2025-10-25 21:40:19.149217152 +0000
+++ b/arch/arm64/boot/dts/rockchip/rk3566-rgb30-linux.dts	2025-10-25 21:39:50.629236871 +0000
@@ -19,6 +19,7 @@
 
 	adc_keys: adc-keys {
 		compatible = "adc-keys";
+		wakeup-source;
 		io-channels = <&saradc 0>;
 		io-channel-names = "buttons";
 		keyup-threshold-microvolt = <1800000>;
@@ -47,6 +48,7 @@
 
 	keys: gpio-keys {
 		compatible = "gpio-keys";
+		wakeup-source;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pwr_key>;
 		autorepeat;
@@ -62,13 +64,14 @@
 			label = "volume down";
 		};
 
-/*		power-key {
+		power-key {
 			wakeup-source;
 			gpios = <&gpio0 5 GPIO_ACTIVE_LOW>;
 			linux,code = <116>;
 			label = "power";
+			enable-method = "psci";
 		};
-*/							
+							
 			
 	};	
         leds: gpio-leds {
@@ -299,6 +302,13 @@
 	};
 };
 
+&rockchip_suspend {
+    rockchip,sleep-debug-en = <0x01>;
+    rockchip,sleep-mode-config = <0x5ec>;
+    rockchip,wakeup-config = <0x10>;
+    status = "okay";
+};
+
 &combphy1_usq {
 	status = "okay";
 };
@@ -602,4 +612,8 @@
 &vcc3v3_lcd0_n {
 	gpio = <&gpio0 RK_PC2 GPIO_ACTIVE_HIGH>;
 	enable-active-high;
+	regulator-state-mem {
+        regulator-on-in-suspend;  // Change to keep powered
+        regulator-suspend-microvolt = <0x325aa0>;
+    };
 };
