--- a/src/render/opengles2/SDL_render_gles2.c	2025-06-02 16:38:50.000000000 +0000
+++ b/src/render/opengles2/SDL_render_gles2.c	2025-12-01 19:10:08.660282612 +0000
@@ -2032,6 +2032,16 @@ static int GLES2_UnbindTexture(SDL_Rende
 
 static int GLES2_CreateRenderer(SDL_Renderer *renderer, SDL_Window *window, Uint32 flags)
 {
+    // Rotation matrix
+    float scale = 640;
+    GLfloat rotmatrix[4][4] = {{0, 1, 0 , 0},
+                               {1, 0, 0,  0},
+                               {0,  0, 1,  0},
+                               {-scale, 0, 0, scale}};
+    SDL_Rect viewport_2;
+    SDL_Rect targetrect;
+    SDL_Point origin;
+
     GLES2_RenderData *data = NULL;
     Uint32 window_flags = 0; /* -Wconditional-uninitialized */
     GLint window_framebuffer;
@@ -2197,8 +2207,8 @@ static int GLES2_CreateRenderer(SDL_Rend
 
     data->drawstate.blend = SDL_BLENDMODE_INVALID;
     data->drawstate.clear_color = 0xFFFFFFFF;
-    data->drawstate.projection[3][0] = -1.0f;
-    data->drawstate.projection[3][3] = 1.0f;
+
+    memcpy(data->drawstate.projection, rotmatrix, 4*4*sizeof(GLfloat));
 
     GL_CheckError("", renderer);
 
--- a/src/render/SDL_render.c	2025-01-01 15:47:53.000000000 +0000
+++ b/src/render/SDL_render.c	2025-12-01 19:10:08.657282615 +0000
@@ -3428,13 +3428,36 @@ int SDL_RenderCopy(SDL_Renderer *rendere
 {
     SDL_FRect dstfrect;
     SDL_FRect *pdstfrect = NULL;
-    if (dstrect) {
-        dstfrect.x = (float)dstrect->x;
-        dstfrect.y = (float)dstrect->y;
-        dstfrect.w = (float)dstrect->w;
-        dstfrect.h = (float)dstrect->h;
-        pdstfrect = &dstfrect;
-    }
+    SDL_Rect targetrect;
+    SDL_Point origin;
+    
+    // Original size, not landscape oriented
+    int ros_w;
+    int ros_h;
+    SDL_Rect viewport;
+    SDL_Rect viewport_2;
+    SDL_GetRendererOutputSize(renderer, &ros_w, &ros_h);
+//    SDL_RenderGetViewport(renderer, &viewport);
+    // probar a invertir el viewport en lugar de usar el logical size
+    viewport_2.h = 640;//ros_h;
+    viewport_2.w = 480;//ros_w;
+    viewport_2.x = 0;
+    viewport_2.y = 0;
+    SDL_RenderSetViewport(renderer, &viewport_2);
+    if(dstrect) { 
+        if(renderer->scale.x > 1.0) {
+        dstfrect.x = -160;//320 * (1 - renderer->scale.x);
+        dstfrect.y = 0 * (1 - renderer->scale.y);
+        dstfrect.w = 240;//((float) ros_w) * renderer->scale.x;
+        dstfrect.h = 320;//((float) ros_h - 160) * renderer->scale.y;
+        } else { // assume scale 1.0
+            dstfrect.x = -(float)ros_w*renderer->scale.x;//319-320.0f * renderer->scale.x;//*3.3;
+            dstfrect.y = 0;//320-240.0f * renderer->scale.y;//*3.3;
+            dstfrect.w = 960;//((float) ros_w) * renderer->scale.x;//*3.3;
+            dstfrect.h = 960;//((float) ros_h - 160) * renderer->scale.y*3.3;
+        } 
+        SDL_Log("DSTRECT: %.1f, %.1f, %.1f, %.1f, SCALE: %.1f, %.1f, ROS: %d, %d\n", dstfrect.x, dstfrect.y, dstfrect.w, dstfrect.h, renderer->scale.x, renderer->scale.y, ros_w, ros_h);
+    }
     return SDL_RenderCopyF(renderer, texture, srcrect, pdstfrect);
 }
 
