diff --git a/yabause/src/retro_arena/main.cpp b/yabause/src/retro_arena/main.cpp
index 39029508..81e0ce7e 100644
--- a/yabause/src/retro_arena/main.cpp
+++ b/yabause/src/retro_arena/main.cpp
@@ -295,9 +295,9 @@ int main(int argc, char** argv)
   if (stat(home_dir.c_str(), &st) == -1) {
     mkdir(home_dir.c_str(), 0700);
   }  
-  std::string bckup_dir = "/run/muos/storage/save/file/YabaSanshiro-Ext/backup.bin";
+  std::string bckup_dir = home_dir + "backup.bin";
   strcpy( buppath, bckup_dir.c_str() );
-  strcpy( s_savepath, "/run/muos/storage/save/state/YabaSanshiro-Ext" );
+  strcpy( s_savepath, home_dir.c_str() );
   g_keymap_filename = home_dir + "keymapv2.json";
 
   std::string current_exec_name = argv[0]; // Name of the current exec program
