diff --git a/yabause/src/CMakeLists.txt b/yabause/src/CMakeLists.txt
index f418fbdc..12345678 100644
--- a/yabause/src/CMakeLists.txt
+++ b/yabause/src/CMakeLists.txt
@@ -1032,9 +1032,9 @@ endif(YAB_WANT_VULKAN)
 
 #-----------------------------------------------------------------------
 # libCHD
 include( ${CMAKE_SOURCE_DIR}/CMake/Packages/external_libchdr.cmake )
-include( ${CMAKE_SOURCE_DIR}/CMake/Packages/external_zlib.cmake )
+#include( ${CMAKE_SOURCE_DIR}/CMake/Packages/external_zlib.cmake )
 #include_directories(${zlib_INCLUDE_DIR})
 add_dependencies(yabause libchdr)
 #add_dependencies(yabause zlib)
 
 
 