diff --git a/yabause/src/retro_arena/CMakeLists.txt b/yabause/src/retro_arena/CMakeLists.txt
index 12345678..87654321 100644
--- a/yabause/src/retro_arena/CMakeLists.txt
+++ b/yabause/src/retro_arena/CMakeLists.txt
@@ -25,6 +25,7 @@ include( ${CMAKE_SOURCE_DIR}/CMake/Packages/external_libpng.cmake )
 # nlohmann/json
 #  * sets EXTERNAL_JSON_INCLUDE_DIR 
 
+if(FALSE) # Disable external JSON download
 ExternalProject_Add(
      Json
      PREFIX ${CMAKE_CURRENT_BINARY_DIR}/Json
@@ -43,6 +44,7 @@ add_custom_target(external-Json-update
      WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Json/src/Json
      COMMAND ${GIT_EXECUTABLE} pull
      DEPENDS Json)
+endif() # Disable external JSON download
 # set include directory
 set(EXTERNAL_JSON_INCLUDE_DIR "${CMAKE_CURRENT_BINARY_DIR}/Json/src/Json/single_include")
 
@@ -66,5 +68,5 @@ target_link_libraries(yabause-retro-arena yabause  ${Boost_LIBRARIES} pugixml $
 target_link_libraries( yabause-retro-arena ${LIBCHDR_LIBRARIES} )
 target_link_libraries( yabause-retro-arena ${zlib_STATIC_LIBRARIES} )
-add_dependencies(yabause-retro-arena Json png zlib libchd )
+#add_dependencies(yabause-retro-arena Json png zlib libchd )
 yab_port_success(yabause-retro-arena)
 set_target_properties(yabause-retro-arena PROPERTIES OUTPUT_NAME yabasanshiro)
 