--- a/yabause/src/retro_arena/CMakeLists.txt
+++ b/yabause/src/retro_arena/CMakeLists.txt
@@ -45,11 +45,15 @@ set(PORT_INCLUDE_DIRS ${EXTERNAL_JSON_INCLUDE_DIR} ${SDL2_INCLUDE_DIR} ${PNG_IN
 set(PORT_LIBRARIES ${SDL2_LIBRARY} ${PNG_LIBRARIES} ${ZLIB_LIBRARIES} )
 
 set(GLES_MALI_LIBS
-  "/muos-sdk/aarch64-buildroot-linux-gnu/sysroot/usr/lib64/libmali.so.0"
-  "/muos-sdk/aarch64-buildroot-linux-gnu/sysroot/usr/lib/libGLESv2.so"
-  "/muos-sdk/aarch64-buildroot-linux-gnu/sysroot/usr/lib/libEGL.so"
+  "mali"
+  "GLESv2"
+  "EGL"
 )
 
+# Add OpenGL ES include directories from buildroot staging
+include_directories("${CMAKE_SYSROOT}/usr/include/GLES2")
+include_directories("${CMAKE_SYSROOT}/usr/include/EGL")
+
 message("Nlohmann/json dir =" ${EXTERNAL_JSON_INCLUDE_DIR})
 
 set(yabause_ra_SOURCES