--- a/yabause/src/ygl.h
+++ b/yabause/src/ygl.h
@@ -43,12 +43,15 @@ extern "C" {
 #endif
 
-#if defined(HAVE_LIBGL) || defined(__ANDROID__) || defined(IOS) || defined(NX)
+#if defined(HAVE_LIBGL) || defined(__ANDROID__) || defined(IOS) || defined(NX) || (defined(__linux__) && defined(_OGLES3_))
 
 #if defined(__LIBRETRO__) && !defined(_USEGLEW_)
     #include <glsym/glsym.h>
     #include <glsm/glsm.h>
 #elif defined(__ANDROID__)
     #include <GLES3/gl31.h>
     #include <EGL/egl.h>
+#elif defined(__linux__) && defined(_OGLES3_)
+    #include <GLES3/gl31.h>
+    #include <EGL/egl.h>
 
 
 #define GL_GEOMETRY_SHADER                0x8DD9
