--- a/yabause/src/retro_arena/main.cpp
+++ b/yabause/src/retro_arena/main.cpp
@@ -450,7 +450,7 @@ int yabauseinit()
   VIDCore->Resize(0,0,width,height,1,p->getInt("Aspect rate",0));
   delete p;
   
-  SDL_GL_MakeCurrent(wnd,nullptr);
+  SDL_GL_MakeCurrent(wnd,glc); // Keep context bound for initialization
 #if defined(__RP64__) || defined(__N2__)
   YabThreadSetCurrentThreadAffinityMask(0x4);
 #else
